micromamba/tests/test_create.py

```diff
@@ -161,6 +161,55 @@ def test_env_logging_overhead_regression(tmp_home, tmp_root_prefix, tmp_path):
     assert res["success"]
 
 
+@pytest.mark.parametrize("shared_pkgs_dirs", [True], indirect=True)
+def test_env_create_pypi(tmp_home, tmp_root_prefix, tmp_path):
+    env_prefix = tmp_path / "myenv"
+
+    create_spec_file = tmp_path / "env-pypi-pkg-test.yaml"
+
+    shutil.copyfile(__this_dir__ / "env-pypi-pkg-test.yaml", create_spec_file)
+
+    res = helpers.create("-p", env_prefix, "-f", create_spec_file, "-y", "--json")
```

Comment:
The tests should be in `test_env.py` instead because we are testing the subcommands of the env command.
That being said, I'm wondering if this is testing the right thing, I think we should use `helpers.run_env(*cmd)` instead, see `test_env_update` for reference.